inst/sql/sql_server/migrations/Migration_1-v3_1_0_time_id.sql

```diff
@@ -0,0 +1,25 @@
+-- Database migrations for version 3.1.0
+-- This migration updates the schema:
+ -- 1. to store the cohort diagnostics version
+ -- 2. converts NULL time_ids in temporal_covariate to 0
+ -- 3. Add a migrations table for supporting database migrations
+
+{DEFAULT @package_version = package_version}
+{DEFAULT @migration = migration}
+{DEFAULT @temporal_covariate_value = temporal_covariate_value}
+{DEFAULT @temporal_covariate_value_dist = temporal_covariate_value_dist}
+{DEFAULT @temporal_covariate_value_dist = temporal_covariate_value_dist}
+{DEFAULT @incidence_rate = incidence_rate}
+{DEFAULT @table_prefix = ''}
+
+-- Create table indicating version number of ddl
+DROP TABLE IF EXISTS @database_schema.@table_prefix@package_version;
+
+--HINT DISTRIBUTE ON RANDOM
+CREATE TABLE @database_schema.@table_prefix@package_version (
+    version_number VARCHAR PRIMARY KEY
```

Comment:
This becomes varchar(MAX) in some platforms, and I wonder if you should apply a max size (like varchar(50) on this.